engine/gamesys/src/gamesys/components/comp_gui.cpp

```diff
@@ -2906,21 +2910,34 @@ namespace dmGameSystem
 
         uint64_t index = pit->m_Next++;
 
-        const char* properties_common[] = { "type", "custom_type", "id", "pivot" };
+        const StrView properties_common[] = {
+            STR_VIEW_LITERAL("type"),
+            STR_VIEW_LITERAL("custom_type"),
+            STR_VIEW_LITERAL("id"),
+            STR_VIEW_LITERAL("pivot"),
+        };
```

Comment:
I don't think the code is improved by this change.
Also, I think the readability suffers a lot, for this code that is also only used when debugging.

Reply:
Understood. I take the closing of this PR as a rejection of the proposal altogether. What's the process for the associated task #9743? Should I as the author close it, will you close it on your Monday meeting, or would you normally do it when closing the PR but simply forgot this time?